pkg/common/cns-lib/vsphere/hostsystem.go

```diff
@@ -95,15 +95,22 @@ type VsanHostCapacity struct {
 	HostMoID         string
 }
 
+type DiskHealth struct {
+	HealthFlags  int   `json:"healthFlags,omitempty"`
+	HealthReason int   `json:"healthReason,omitempty"`
+	TimeStamp    int64 `json:"timestamp,omitempty"`
+}
+
 // VsanPhysicalDisk reflects the fields of JSON structure emitted by the
 // VsanInternalSystem.QueryPhysicalVsanDisks API that we care about.
 type VsanPhysicalDisk struct {
-	IsSSD            int    `json:"isSsd,omitempty"`
-	SsdUUID          string `json:"ssdUuid,omitempty"`
-	Capacity         int64  `json:"capacity,omitempty"`
-	CapacityReserved int64  `json:"capacityReserved,omitempty"`
-	CapacityUsed     int64  `json:"capacityUsed,omitempty"`
-	IsAllFlash       int    `json:"isAllFlash,omitempty"`
+	IsSSD            int        `json:"isSsd,omitempty"`
+	SsdUUID          string     `json:"ssdUuid,omitempty"`
+	Capacity         int64      `json:"capacity,omitempty"`
+	CapacityReserved int64      `json:"capacityReserved,omitempty"`
+	CapacityUsed     int64      `json:"capacityUsed,omitempty"`
+	IsAllFlash       int        `json:"isAllFlash,omitempty"`
+	DiskHealth       DiskHealth `json:"disk_health,omitempty"`
```

Comment:
Can you please change either the type name or the member name? I think it will be easier to read if they are different.

Reply:
Updated the field name to `Disk_Health`